docs/io/unified.rst

```diff
@@ -36,12 +36,25 @@ The example below shows how to read a table in the specialized DAOphot format an
 it back to FITS format. Notice that FITS is a format where the interface recognizes the
 format automatically from the file name, so the ``format`` argument is not needed.
 
-.. doctest-skip::
+.. testsetup::
+    >>> import os
+    >>> with open('photometry.dat', 'w') as f: # doctest: +IGNORE_OUTPUT
+    ...     f.write("#N ID    XCENTER   YCENTER\n")
+    ...     f.write("#U ##    pixel     pixel \n")
+    ...     f.write("#F %-9d  %-10.3f   %-10.3f\n")
+    ...     f.write("#\n")
+    ...     f.write("14       138.538   256.405\n")
+    ...     f.write("18       18.114    280.170\n")
 
     >>> from astropy.table import Table
```

Comment:
In the current documentation - https://astropy--18524.org.readthedocs.build/en/18524/io/unified.html - the whole example is not shown!!

The difference with other examples is that here you do have the code indented, which in principle is nicer... It may need an `::` on the line above...